hummingbot/strategy/liquidity_mining/data_types.py

```diff
@@ -1,8 +1,16 @@
 #!/usr/bin/env python
+
+"""
+Liquidity mining data types...
+"""
+
 from decimal import Decimal
 
 
 class PriceSize:
+    """
+    What is this? price of asset and amount of asset to sell?
```

Comment:
A simple data structure class for order price and order size, probably can be replace with `dataclass`

Reply:
Resolved in commit https://github.com/CoinAlpha/hummingbot/pull/3440/commits/b35ab311e01d62f7b1b468e7d6fb855932b78fa4